static/nodejs_server/ranking_tile.ts

```diff
@@ -107,8 +107,11 @@ function getRankingUnitResult(
   useChartUrl: boolean,
   apiRoot: string
 ): TileResult {
+  const points = isHighest
+    ? rankingGroup.points.slice().reverse()
```

Comment:
Will this sort by RankingPoint.value?

Reply:
yes, rankingGroup.points is already sorted by value, but from lowest to highest